client-js/package.json

```diff
@@ -39,7 +38,8 @@
   },
   "dependencies": {
     "@ethersproject/bignumber": "^5.7.0",
-    "casper-js-sdk": "2.10.1",
+    "@make-software/ces-js-parser": "^1.1.0",
```

Comment:
Are you sure you want the sdk to pull any new version, because if there is any breaking change you can't control the update for this release . So I am just checking we don't want to fix the version to 1.1.0 for this release. Please check that with Make. (not 100 % sure but I see you pin the version of the sdk too)

Reply:
yeah its pinned right now, it important especially that they are using `2.11` too